test/functional/rpc_signrawtransactionwithkey.py

```diff
@@ -4,9 +4,6 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
8c64048c443cddce201b6d54184b32f3894ab806 can be squashed together with the first commit